### ISSUE TITLE
Tabs don't process shortcode

### ISSUE THREAD

**Issue by immeemz** [ThemeFuse/Unyson#3586]
Hi, I created some Tabs on my homepage and placed some shortcode in the text editor of each tab.  When I view the site, it just spits back out the shortcode instead of displaying what the shortcode is meant to display. Elsewhere in the site, the shortcode works fine, including within Unyson "Text Block".  Just the Tabs don't work. Is there a fix?

**Comment by riponhossainchopol** [comment]
without code how can we help?

**Comment by immeemz** [comment]
Hi, thanks for responding!  Well, it's any and all shortcodes that don't work in the tabs. It doesn't matter which shortcode it is, none of them work.  However, one of the shortcodes in question is [etsy-shop shop_name="FlambeauCreation" section_id="24627860"].  I also can't use one from Contact Form 7, and the code is [contact-form-7 id="4955" title="Contact form page"].  You can view them all not working at this page:  https://flambeaucreations.com/tabs-test/

Thank you!  
--

**Comment by immeemz** [comment]
Yes, I am using that shortcode on this page:  https://flambeaucreations.com/contact/  The shortcode is in a Unysom "Text block".  The Text Block will process the shortcode correctly, but the Tabs and Accordions do not.  I am also using the other (Etsy) shotcode in a Text Block on the homepage of the website (https://flambeaucreations.com/) , and it works fine.  Thanks for any help you can give. :o)

**Comment by emillerphx** [comment]
I looked into the code loaded on our server. The do_shortcode is there.  I'm stumped.

**Comment by emillerphx** [comment]
the clue above from danyj got me there.  
I went into ./wp-content/themes/solarify/framework-customizations/extensions/shortcodes/shortcodes/tabs/views
Edited views.php
the do_shortcode was commented out. I removed the comment and it works, sort of.  The shortcode itself shows up as text. I'm going to set the text color to the same as the background till I have time to come back and try and fix it.
